### ISSUE TITLE
EE3

### ISSUE THREAD

**Issue by ppappila** [Electrical-Age/ElectricalAge#58]
EE3 does not get along with your mod....hmmmm

**Comment by Dolu1990** [comment]
Hi,
Can you be more accurate ? there is a mod incompatibilites => crash ?
What do EE3 ? (never played with)

**Comment by Techokami** [comment]
Equivalent Exchange 3, it's a mod that lets you transmute blocks and items into a "currency" of matter energy that can then be turned into other items. Its previous incarnation was infamous for being extremely game breaking (generate diamonds out of nothing at all!)

**Comment by ppappila** [comment]
ok here you go...here is the crash report with you mod installed. Also  I made just for you a separate instance with just ee3 and your mod, no other mods where installed no ever NEI.

---- Minecraft Crash Report ----
// Don't be sad, have a hug! <3

Time: 21/07/14 6:38 PM
Description: Initializing game

java.lang.NullPointerException: Initializing game
    at net.minecraft.item.ItemStack.func_77960_j(ItemStack.java:215)
    at com.pahimar.ee3.exchange.WrappedStack.hashCode(WrappedStack.java:309)
    at java.util.AbstractList.hashCode(AbstractList.java:541)
    at java.util.HashMap.hash(HashMap.java:338)
    at java.util.HashMap.put(HashMap.java:611)
    at java.util.HashSet.add(HashSet.java:219)
    at com.google.common.collect.AbstractMapBasedMultimap.put(AbstractMapBasedMultimap.java:195)
    at com.google.common.collect.AbstractSetMultimap.put(AbstractSetMultimap.java:121)
    at com.google.common.collect.HashMultimap.put(HashMultimap.java:49)
    at com.pahimar.ee3.recipe.RecipesVanilla.init(RecipesVanilla.java:52)
    at com.pahimar.ee3.recipe.RecipesVanilla.getVanillaRecipes(RecipesVanilla.java:26)
    at com.pahimar.ee3.recipe.RecipeRegistry.init(RecipeRegistry.java:30)
    at com.pahimar.ee3.recipe.RecipeRegistry.<init>(RecipeRegistry.java:24)
    at com.pahimar.ee3.recipe.RecipeRegistry.getInstance(RecipeRegistry.java:82)
    at com.pahimar.ee3.EquivalentExchange3.postInit(EquivalentExchange3.java:99)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:513)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:208)
    at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:187)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:118)
    at cpw.mods.fml.common.Loader.initializeMods(Loader.java:694)
    at cpw.mods.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:288)
    at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:541)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:867)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:134)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:273)
    at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:363)
    at org.multimc.EntryPoint.listen(EntryPoint.java:165)
    at org.multimc.EntryPoint.main(EntryPoint.java:54)

## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
    at net.minecraft.item.ItemStack.func_77960_j(ItemStack.java:215)
    at com.pahimar.ee3.exchange.WrappedStack.hashCode(WrappedStack.java:309)
    at java.util.AbstractList.hashCode(AbstractList.java:541)
    at java.util.HashMap.hash(HashMap.java:338)
    at java.util.HashMap.put(HashMap.java:611)
    at java.util.HashSet.add(HashSet.java:219)
    at com.google.common.collect.AbstractMapBasedMultimap.put(AbstractMapBasedMultimap.java:195)
    at com.google.common.collect.AbstractSetMultimap.put(AbstractSetMultimap.java:121)
    at com.google.common.collect.HashMultimap.put(HashMultimap.java:49)
    at com.pahimar.ee3.recipe.RecipesVanilla.init(RecipesVanilla.java:52)
    at com.pahimar.ee3.recipe.RecipesVanilla.getVanillaRecipes(RecipesVanilla.java:26)
    at com.pahimar.ee3.recipe.RecipeRegistry.init(RecipeRegistry.java:30)
    at com.pahimar.ee3.recipe.RecipeRegistry.<init>(RecipeRegistry.java:24)
    at com.pahimar.ee3.recipe.RecipeRegistry.getInstance(RecipeRegistry.java:82)
    at com.pahimar.ee3.EquivalentExchange3.postInit(EquivalentExchange3.java:99)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:513)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:208)
    at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:187)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:118)
    at cpw.mods.fml.common.Loader.initializeMods(Loader.java:694)
    at cpw.mods.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:288)
    at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:541)

-- Initialization --
Details:
Stacktrace:
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:867)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:134)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:273)
    at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:363)
    at org.multimc.EntryPoint.listen(EntryPoint.java:165)
    at org.multimc.EntryPoint.main(EntryPoint.java:54)

-- System Details --
Details:
    Minecraft Version: 1.7.10
    Operating System: Linux (amd64) version 3.13.0-24-generic
    Java Version: 1.8.0_11, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 548778776 bytes (523 MB) / 801112064 bytes (764 MB) up to 954728448 bytes (910 MB)
    JVM Flags: 2 total; -Xms512m -Xmx1024m
    AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP v9.05 FML v7.10.24.1187 Minecraft Forge 10.13.0.1187 5 mods loaded, 5 mods active
    mcp{9.05} [Minecraft Coder Pack](minecraft.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized
    FML{7.10.24.1187} [Forge Mod Loader](forge-1.7.10-10.13.0.1187.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized
    Forge{10.13.0.1187} [Minecraft Forge](forge-1.7.10-10.13.0.1187.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized
    Eln{21} [Electrical Age](ElectricalAge_BETA-1.7_r21.jar) Unloaded->Constructed->Pre-initialized->Initialized->Post-initialized
    EE3{0.2.281} [Equivalent Exchange 3](EquivalentExchange3-1.7.10-0.2.281.jar) Unloaded->Constructed->Pre-initialized->Initialized->Errored
    Launched Version: 1.7.10
    LWJGL: 2.9.1
    OpenGL: GeForce GTX 660 Ti/PCIe/SSE2 GL version 4.4.0 NVIDIA 331.38, NVIDIA Corporation
    GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

```
Is Modded: Definitely; Client brand changed to 'fml,forge'
Type: Client (map_client.txt)
Resource Packs: []
Current Language: English (US)
Profiler Position: N/A (disabled)
Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
Anisotropic Filtering: Off (1)
```

**Comment by Dolu1990** [comment]
Hi,
I have investigate. It's a EE3 bug, that was fixed into EE3 issue #705. 
Update EE3 and all is ok.
Any way, thanks for taking time to put issue + electrical age + EE3 sandbox setup :)
